packages/app-desktop/gui/NoteEditor/NoteBody/CodeMirror/CodeMirror.tsx

```diff
@@ -614,6 +614,7 @@ function CodeMirror(props: NoteBodyEditorProps, ref: any) {
 				resourceInfos: props.resourceInfos,
 				contentMaxWidth: props.contentMaxWidth,
 				mapsToLine: true,
+				useCustomPdfViewer: true,
```

Comment:
Does that mean we can easily switch between native and custom viewer? Did you try to setting this false, and does it still work?

We indeed need this in case there's a problem. That way we can switch off the new renderer which gives us time to investigate the issue. So please make sure this flag works correctly and test it regularly.

Reply:
Yes, it is used to toggle between the two viewers. Currently I have hardcoded it as true but in future I'm planning to add a setting for it in the preferences, if there's a need or time permits.